bigtable/admin.go

```diff
@@ -275,6 +278,35 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// UpdateTable updates a table in the instance from the given configuration.
+// Only deletion protection can be updated at this moment
+func (ac *AdminClient) UpdateTable(ctx context.Context, conf *TableConf) (updated bool, err error) {
+	if conf.TableID == "" {
+		return false, errors.New("TableID is required")
+	}
+	if conf.Families != nil || conf.SplitKeys != nil {
+		return false, errors.New("Only deletion protection field can be updated")
```

Comment:
NIT: Go style is to write messages as all lower-case.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings